.babelrc

```diff
@@ -12,7 +12,6 @@
     "lodash"
   ],
   "presets": [
-    "es2015",
```

Comment:
This should stay, right?

Reply:
Babel is pushing people to the `preset-env` setup and actually warning people when they install the `es2015` preset. That correctly transforms get pulled in based on our build targets.